### DISCUSSION TITLE
Arreglo del release.yml

### DISCUSSION THREAD

**Discussion by laurolmer** [perealberto/decide-single-zambrano#32] [General]
Chicos me he dado cuenta de que el código del release.yml no funcionaba y daba errores cuando se ejecutaba, voy a revisarlo y corregirlo.
De paso, intento crear una tag y un release para verificar que funciona.
¿Cómo lo veis?

**Comment by MarioArocaPaez** [comment]
Me parece bien, proceda y mantenganos al tanto de sus actualizaciones compañera
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/47d5dc1f-22f1-43ff-96de-dc80a6a40810)

**Comment by perealberto** [comment]
A mí me parece bien también @laurolmer. Si necesitas ayuda lo podemos ver en llamada. Avísanos :cactus:.
